app/services/radical_twilio.rb

```diff
@@ -0,0 +1,40 @@
+class RadicalTwilio
+  def self.send_sms(from:, to:, message:)
+    client.messages.create(from: from, to: to, body: message)
+  end
+
+  def self.send_robocall(from:, to:, url:)
+    client.calls.create(from: from, to: to, url: URI::encode(url))
+  end
+
+  def self.client
+    Twilio::REST::Client.new
+  end
+
+  def self.twilio_enabled?
+    defined?(Twilio).present? && Company.has_attribute?(:twilio_phone_numbers) && client.present?
```

Comment:
I was thinking we could just check if the env vars exist, or was this better for some reason?